src/worker/handlers.rs

```diff
@@ -76,7 +76,7 @@ macro_rules! handle_store {
             vec![MimeType::TextPlainUtf8.to_string(), MimeType::Utf8String.to_string()],
             move |event, _| {
                 if let $event_ty::Send { mut pipe, .. } = event {
-                    write!(pipe, "{}", $contents).unwrap();
+                    let _ = write!(pipe, "{}", $contents);
```

Comment:
It's maybe worth adding a comment why this error is to be ignored, maybe?

Reply:
well, it doesn't really matter if we can't write? Like reader decided to close things, so it doesn't matter if we fail due to broken pipe?

Reply:
Then, let's just leave a comment saying that if this write fails, that means the other side has closed the pipe and there is nothing more to do.